modules/rostests/apitests/ntdll/RtlDosSearchPath_Ustr.c

```diff
@@ -210,4 +213,23 @@ START_TEST(RtlDosSearchPath_Ustr)
     ok_eq_pointer(FullNameOut, NULL);
     ok_eq_ulong(FilePartSize, 0UL);
     ok_eq_ulong(LengthNeeded, 0UL);
+
+    /* Buffer overflow test
+     * length(longDirName) + length(longFileName) + length(ext) = MAX_PATH
+     */
+    RtlInitUnicodeString(&PathString, longDirName);
```

Comment:
Why not putting the test strings directly there, instead of having them (much) above in the code?

Reply:
Done.